core/src/server/host_filtering/mod.rs

```diff
@@ -0,0 +1,6 @@
+//! Host filtering.
```

Comment:
Is the idea that in a followup PR we might be able to move this stuff into a tower middleware?

Reply:
yeah, if it will become cleaner

it's so little code now so I'm okay keeping it here as well.